graphql_compiler/scripts/generate_test_sql/utils.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import random
+from uuid import UUID
+
+import six
+
+
+CREATE_VERTEX = 'create vertex '
+CREATE_EDGE = 'create edge '
+SEPARATOR = '__'
+
+
+def get_uuid():
+    """Return a pseudorandom uuid."""
+    return str(UUID(int=random.randint(0, 2**128 - 1)))  # nosec
+
+
+def get_random_net_worth():
+    """Return a pseudorandom uuid."""
+    return int(1e3*random.random() * 100) / 100.0  # nosec
```

Comment:
spacing between operators -- also, any particular reason why `1e3` and `100` are separate?